src/Containers/MassTransit.ExtensionsDependencyInjectionIntegration/DependencyInjectionTestingExtensions.cs

```diff
@@ -1,13 +1,19 @@
 namespace MassTransit.Testing
 {
     using System;
+    using Automatonymous;
     using ExtensionsDependencyInjectionIntegration;
     using Microsoft.Extensions.DependencyInjection;
+    using Registration;
+    using Saga;
 
 
     public static class DependencyInjectionTestingExtensions
     {
-        public static IServiceCollection AddInMemoryTestHarness(this IServiceCollection services,
+        /// <summary>
+        /// Add the In-Memory test harness to the container, and configure it using the callback specified.
+        /// </summary>
+        public static IServiceCollection AddMassTransitInMemoryTestHarness(this IServiceCollection services,
```

Comment:
Why does not create factory as we did for other transports?

Reply:
Because the bus is already created inside the test harness, I didn't want to rewrite any of that stuff. It might be convertible to a factory, but it will still just return the buscontrol.